bundles/paikkatietoikkuna/tiles3d/plugin/Tiles3DModelBuilder.js

```diff
@@ -0,0 +1,15 @@
+const Style = Oskari.clazz.get('Oskari.mapframework.domain.Style');
+
+export default class Tiles3DModelBuilder {
+    parseLayerData (layer, mapLayerJson, maplayerService) {
+        if (mapLayerJson.options.styles) {
```

Comment:
Add a check for options to be present in mapLayerJson before referencing styles.